Signal/src/ViewControllers/ConversationView/ConversationStyle.swift

```diff
@@ -34,7 +34,6 @@ public class ConversationStyle: NSObject {
     @objc public var contentWidth: CGFloat = 0
 
     // viewWidth - (gutterfullWidthGutterLeadingLeading + fullWidthGutterTrailing)
-    // TODO: Is this necessary?
```

Comment:
I suggest we move this class to SignalMessaging and pull the contents of `OWSMessagesBubbleImageFactory`/`OWSMessagesBubbleColors` into this class.

Reply:
Sure - is it OK if I include that in my work, or is that blocking something you're doing?